app/controllers/donations_controller.rb

```diff
@@ -18,40 +18,14 @@ def print
   def index
     setup_date_range_picker
 
-    @donations = current_organization.donations
-                                     .includes(:storage_location, :donation_site, :product_drive, :product_drive_participant, :manufacturer, line_items: [:item])
-                                     .order(created_at: :desc)
-                                     .class_filter(filter_params)
-                                     .during(helpers.selected_range)
-    @item_categories = current_organization.item_categories.pluck(:name).uniq
-    @paginated_donations = @donations.page(params[:page])
-
-    @product_drives = current_organization.product_drives.alphabetized
-    @product_drive_participants = current_organization.product_drive_participants.alphabetized
-
-    # Are these going to be inefficient with large datasets?
-    # Using the @donations allows drilling down instead of always starting with the total dataset
-    @donations_quantity = @donations.collect(&:total_quantity).sum
-    @paginated_donations_quantity = @paginated_donations.collect(&:total_quantity).sum
-    @total_value_all_donations = total_value(@donations)
-    @paginated_in_kind_value = total_value(@paginated_donations)
-    @total_money_raised = total_money_raised(@donations)
```

Comment:
Looks like several of these helpers should also be removed from this controller now that they live in the view